smk/assembly.smk

```diff
@@ -249,14 +263,29 @@ rule correct_spadeshammer:
         ) >& {log}
         """
 
+def get_corrected_runs_for_sample(wildcards):
+    if wildcards.illumina in merged_illumina_samples:
+        files = list()
+        for r in merged_illumina_samples[wildcards.illumina].split('+'):
+            rep = r.strip()
+            files += expand("{wd}/{omics}/5-corrected-runs/{rep}/{run}.{pair}.fq.gz",
+                                    wd = wildcards.wd,
+                                    omics = wildcards.omics,
+                                    rep = rep,
+                                    run = get_runs_for_sample(wildcards.wd, wildcards.omics, rep),
+                                    pair = wildcards.pair)
+    else:
+        files = expand("{wd}/{omics}/5-corrected-runs/{illumina}/{run}.{pair}.fq.gz",
+                                    wd = wildcards.wd,
+                                    omics = wildcards.omics,
+                                    illumina = wildcards.illumina,
+                                    run = get_runs_for_sample(wildcards.wd, wildcards.omics, wildcards.illumina),
+                                    pair = wildcards.pair)
+    return(files)
+
```

Comment:
If the input to spadeshammer are the non-redundant samples, how are the reps' corrected runs made?

Reply:
Input to spadeshammer is always runs, as it corrects run by run.